src/rest_api/classes/laboratory/widgets/alleles.clj

```diff
@@ -16,13 +16,11 @@
                          (let [allele (d/entity db wbvar)]
                            (pace-utils/vmap
                              :sequenced 
-                             (if (= (str (:variation/seqstatus allele))
+                             (if (= (:variation/seqstatus allele)
                                     ":variation.seqstatus/sequenced") "yes" "no")
```

Comment:
Value being compared shouldn't be a string (is it?) (i.e: ":variation.seqstatus/sequenced" _should_ be a keyword/ident not a string)